### PR TITLE
Initial tests configuration

### PR DESCRIPTION
**Pull request by MarioBlazek**
Please enable TravisCI and Codecov integration before merging this PR.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,9 +13,16 @@
         "lolautruche/ez-core-extra-bundle": "^1.0",
         "netgen/ngsymfonytools": "^1.1"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7",
+        "netgen/tagsbundle": "~3.0"
```
Comment (emodric) [inline composer.json]: Please use `^3.0` here.

**Comment by MarioBlazek** [comment]
Code syntax fixed and rebase done.

**Comment by emodric** [comment]
Now the tests need fixing ;)

Can you also add testing on PHP 5.4 and PHP 5.5, since both of those are supported still?

**Comment by emodric** [comment]
Still failing :)

Is Tags Bundle really required in `require-dev` ?

**Comment by emodric** [comment]
Thanks @MarioBlazek !

**Pull request merged**